README.md

```diff
@@ -496,6 +496,18 @@ HostbasedAuthentication: Enables/disables Hostbased authentication. Valid values
 
 - *Default*: 'no'
 
+sshd_ignoreuserknownhosts
+-------------------------
+IgnoreUserKnownHosts: Enables/disables Hostbased authentication. Valid values are 'yes' and 'no'.
```

Comment:
Taking the comment from sshd_config(5) man page and putting in some standard language. Please update with the following.


String for IgnoreUserKnownHosts option in sshd_config. Valid values are 'yes' and 'no. Specifies whether sshd(8) should ignore the user's `~/.ssh/known_hosts` during RhostsRSAAuthentication or HostbasedAuthentication. The default is 'no'.